src/literal.ml

```diff
@@ -11,3 +11,16 @@ external float64_bits : int64 -> t = "caml_binaryen_literal_float64_bits"
 let float32 n = float32_bits @@ Int32.bits_of_float n
 
 external float64 : float -> t = "caml_binaryen_literal_float64"
+
+(* Hacks for Binaryen.js stack allocations, Don't use in binaryen.native *)
+type jsoo =
```

Comment:
This is absolutely terrible, but I couldn't find a way around it while still using Binaryen.js because their `const` implementations use their custom stack.